infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml

```diff
@@ -93,6 +93,9 @@
                 <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
             </formatter>
         </subsystem>
+        <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
+            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```

Comment:
@DavideD , we added scanner to make sure the jars are deployed to Infinispan server